docker/llm/inference/cpu/docker/README.md

```diff
@@ -32,3 +32,13 @@ sudo docker run -itd \
 After the container is booted, you could get into the container through `docker exec`.
 
 To run inference using `BigDL-LLM` using cpu, you could refer to this [documentation](https://github.com/intel-analytics/BigDL/tree/main/python/llm#cpu-int4).
+
+### Use chat.py
+
+chat.py can be used to initiate a conversation with a specified model. The file is under directory '/root'.
```

Comment:
`chat.py` can be used to initiate a conversation with a specified model on Intel CPU.

Add some description that the model is needed to be downloaded by users.

Reply:
added